lib/do_volume/do_create.js

```diff
@@ -178,6 +178,13 @@ do_create.options = [
         type: 'arrayOfBool',
         help: 'Wait for the creation to complete. Use multiple times for a ' +
             'spinner.'
+    },
+    {
+        names: ['wait-timeout'],
+        type: 'positiveInteger',
+        default: 120,
```

Comment:
Good point again, and I agree with all your statements and questions. Infinite default timeout sounds like the right thing to do here.